tests/simulate/test_lammps.py

```diff
@@ -0,0 +1,171 @@
+        pots.pair.num = 11
+
+        return (ens,pots)
+
+    def create_file(self):
+        file_ = self.directory.file('test.data')
+        with open(file_,'w') as f:
+            f.write(('LAMMPS test data\n'
+                     '\n'
+                     '5 atoms\n'
+                     '2 atom types\n'
+                     '\n'
+                     '0.0 10.0 xlo xhi\n'
+                     '0.0 10.0 ylo yhi\n'
+                     '0.0 10.0 zlo zhi\n'
+                     '\n'
+                     'Atoms\n'
+                     '\n'
+                     '1 1 1.0 1.0 1.0\n'
+                     '2 1 3.0 3.0 3.0\n'
+                     '3 2 5.0 5.0 5.0\n'
+                     '4 2 7.0 7.0 7.0\n'
+                     '5 2 9.0 9.0 9.0\n'
+                     '\n'
+                     'Masses\n'
+                     '\n'
+                     '1 0.3\n'
+                     '2 0.1'))
+        return file_
+
+    def test_initialize(self):
+        """Test running initialization simulation operations."""
+        #InitializeFromFile
+        ens,pot = self.ens_pot()
+        file_ = self.create_file()
+        op = relentless.simulate.lammps.InitializeFromFile(filename=file_, neighbor_buffer=0.4)
+        l = relentless.simulate.lammps.LAMMPS(operations=op, quiet=False)
+        sim = l.run(ensemble=ens, potentials=pot, directory=self.directory)
+        pl = lammps.PyLammps(ptr=sim.lammps)
+        self.assertIsNotNone(pl.system)
+
+        #InitializeRandomly
+        op = relentless.simulate.lammps.InitializeRandomly(seed=1, neighbor_buffer=0.4)
+        l = relentless.simulate.lammps.LAMMPS(operations=op, quiet=False)
+        sim = l.run(ensemble=ens, potentials=pot, directory=self.directory)
+        pl = lammps.PyLammps(ptr=sim.lammps)
+        self.assertIsNotNone(pl.system)
+
+    def test_minimization(self):
+        """Test running energy minimization simulation operation."""
+        #MinimizeEnergy
+        ens,pot = self.ens_pot()
+        file_ = self.create_file()
+        op = [relentless.simulate.lammps.InitializeFromFile(filename=file_, neighbor_buffer=0.4),
+              relentless.simulate.lammps.MinimizeEnergy(energy_tolerance=1e-7,
+                                                        force_tolerance=1e-7,
+                                                        max_iterations=1000,
+                                                        dt=0.01)
+             ]
+        l = relentless.simulate.lammps.LAMMPS(operations=op, quiet=False)
+        sim = l.run(ensemble=ens, potentials=pot, directory=self.directory)
+
+    def test_integrators(self):
+        """Test adding and removing integrator operations."""
+        default_fixes = [{'name':''}]
+
+        file_ = self.create_file()
+        init = relentless.simulate.lammps.InitializeFromFile(filename=file_, neighbor_buffer=0.4)
+        l = relentless.simulate.lammps.LAMMPS(operations=init, quiet=False)
+
+        #LangevinIntegrator
```

Comment:
Can you run a test of the Langevin integrator with only one type? That would catch bugs like the one I think I spotted.